apis/r/DESCRIPTION

```diff
@@ -64,10 +67,11 @@ Suggests:
     withr,
     testthat (>= 3.0.0),
     pbmc3k.tiledb,
+    pbmc3k.sce,
```

Comment:
OTOH I would be happy to add the package the one Additional_repositories we already have.

Reply:
Moving conversation to https://github.com/single-cell-data/TileDB-SOMA/pull/1547#discussion_r1290428532